src/provider/hyperkzg.rs

```diff
@@ -1,5 +1,5 @@
 //! This module implements Nova's evaluation engine using `HyperKZG`, a KZG-based polynomial commitment for multilinear polynomials
-//! HyperKZG is based on the transformation from univariate PCS to multilinear PCS in the Gemini paper (section 2.4.2 in https://eprint.iacr.org/2022/420.pdf).
+//! HyperKZG is based on the transformation from univariate PCS to multilinear PCS in the Gemini paper (section 2.4.2 in `<https://eprint.iacr.org/2022/420.pdf>`).
```

Comment:
Is this the correct way to denote URLs in markdown?

Reply:
https://doc.rust-lang.org/beta/rustdoc/lints.html#bare_urls